src/main/java/com/solr/clientwrapper/domain/port/api/kafkaSender.java

```diff
@@ -0,0 +1,7 @@
+package com.solr.clientwrapper.domain.port.api;
+
+public interface kafkaSender {
```

Comment:
Class/interface names should start with capital letter

Reply:
done